src/lotus/mod.rs

```diff
@@ -0,0 +1,42 @@
+pub mod client;
+///! The lotus api to interact with lotus node
+pub mod message;
+
+use anyhow::Result;
+use async_trait::async_trait;
+use cid::Cid;
+use fvm_shared::address::Address;
+use serde::de::DeserializeOwned;
+use std::fmt::Debug;
+
+pub use crate::lotus::client::LotusJsonRPCClient;
+pub use crate::lotus::message::{MpoolPushMessage, MpoolPushMessageInner};
+pub use crate::lotus::message::{
+    ReadStateResponse, StateWaitMsgResponse, WalletKeyType, WalletListResponse,
+};
+
+/// The Lotus client api to interact with the Lotus node.
+#[async_trait]
+pub trait LotusClientApi {
```

Comment:
Please drop "Api" from the name as it adds confusion to the intention. I suggest you rename it to `LotusJsonRpcClient` as suggested before or just `LotusClient`, though the former is more descriptive.